### DISCUSSION TITLE
Openradar Discourse Group

### DISCUSSION THREAD

**Discussion by rcjackson** [openradar/PyDDA#78] [Announcements]
PyDDA user community,

We are moving general use questions for PyDDA to the Openradar Discourse Group where the open radar science community and PyDDA developers can answer any questions you may have. In order to access the Discourse group, just click this [link](https://openradar.discourse.group). Software bugs should still be raised in the issue tracker.

Bobby

**Comment by BenNgHKO** [comment]
Could you please upload the last version(2.0.1) into https://pypi.org/project/pydda/ ?

**Comment by rcjackson** [comment]
Hello. I have fixed the issues with PyDDA’s automatic PyPI release. Version 2.0.2 should reflect all of the latest fixes. I anticipate it being available in conda-forge by tomorrow.

From: BenNgHKO ***@***.***>
Date: Wednesday, October 9, 2024 at 10:17 PM
To: openradar/PyDDA ***@***.***>
Cc: Jackson, Robert ***@***.***>, Author ***@***.***>
Subject: Re: [openradar/PyDDA] Openradar Discourse Group (Discussion #78)
Could you please upload the last version(2. 0. 1) into https: //pypi. org/project/pydda/ ? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread. Message ID: <openradar/PyDDA/repo-discussions/78/comments/10899065@ github. com>
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.

ZjQcmQRYFpfptBannerEnd

Could you please upload the last version(2.0.1) into https://pypi.org/project/pydda/<https://urldefense.us/v3/__https:/pypi.org/project/pydda/__;!!G_uCfscf7eWS!e5nw7DqPqLguVd9aAqIcBFuv5h60L6TmLkg4YCM6eaKn-2A5PWBu3M6iniHC9tSij-hz-qMLwjnhM8YslnFqKOLK$> ?

**Comment by BenNgHKO** [comment]
Hello Jackson, I would like to know about the pydda.retrieval.get_dd_wind_field(), the sequence of the first input param("Grids") will affect the result?
